### DISCUSSION TITLE
feature request to add some form of progress indicator on first launch

### DISCUSSION THREAD

**Discussion by kwoot** [mu-editor/mu#1483] [Ideas & New Features]
I know there is progress as I see my harddisc-led flashing. But newbies to this editor (your main demographic) probably don't know that. Some form of progress indicator on launching for the first time might be a good idea.

Weird. On first execution it hanged. Killing and restarting and working (Ubuntu 20.10).

Anyway, a great addition to get newbies to programming! Excellent work!

**Comment by ZanderBrown** [comment]
The beta version should already be presenting an animated splash screen as soon as possible (AFAIK), how long (roughly) did it take for you?

**Comment by kwoot** [comment]
Yes, it opened a splash screen with some info lines at the bottom. And then only hard-disc and no clue what was happening. No biggy of course, just thinking from the context of a real newbie.
